packages/api/src/tasks/checkAlerts.ts

```diff
@@ -1,4 +1,3 @@
-// @ts-nocheck TODO: Fix When Restoring Alerts
```

Comment:
This file isn't fully ready yet. Just fixed types. Will need to hook chartConfig stuff